src/runtime/test/data-layer-test.ts

```diff
@@ -11,40 +11,33 @@
 import {assert} from '../../platform/chai-web.js';
 import {Arc} from '../arc.js';
 import {FakeSlotComposer} from '../testing/fake-slot-composer.js';
+import {Loader} from '../loader.js';
 import {Schema} from '../schema.js';
 import {EntityType} from '../type.js';
 import {handleFor} from '../handle.js';
+import {CollectionStorageProvider} from '../storage/storage-provider-base.js';
 
-describe('entity', async function() {
+describe('entity', async () => {
   it('can be created, stored, and restored', async () => {
     const schema = new Schema(['TestSchema'], {value: 'Text'});
 
-    const arc = new Arc({slotComposer: new FakeSlotComposer(), id: 'test'});
+    const arc = new Arc({slotComposer: new FakeSlotComposer(), id: 'test', context: null, loader: new Loader()});
     const entity = new (schema.entityClass())({value: 'hello world'});
     assert.isDefined(entity);
-    const storage = await arc.createStore(new EntityType(schema).collectionOf());
-    const handle = handleFor(storage);
+
+    const collectionType = new EntityType(schema).collectionOf();
+    
+    // Get around incompatible types for handleFor()
+    let storage;
+    storage = await arc.createStore(collectionType);
+    const handle = handleFor(storage );
```

Comment:
nit - extra space between `storage` and `);`

Reply:
fixed.  thanks!